gateway/modules/auth/operations.go

```diff
@@ -12,3 +17,22 @@ func (m *Module) Encrypt(value string) (string, error) {
 
 	return utils.Encrypt(m.aesKey, value)
 }
+
+func (m *Module) getFields(ctx context.Context, fields interface{}, args map[string]interface{}) ([]interface{}, error) {
```

Comment:
Can we return array of string here?

Reply:
I suggest you move this function to `helpers.go`

Reply:
[] string? we can but an unnecessary for loop

Reply:
There is no helpers.go in auth package?

Reply:
So lets return [] interface only. Also, make a helpers.go file